docker/CMSRucioClient/scripts/syncUserRoles.py

```diff
@@ -0,0 +1,258 @@
+                    set_approvers = True
+                    break
+            if not set_approvers:
+                print(f"No site manager found for {rse_name}")
+
+
+def set_rse_manager(client, rse_name, site_managers, alt_rse=None):
+    if not alt_rse:
+        alt_rse = rse_name.lower()
+    else:
+        alt_rse = alt_rse.lower()
+    try:
+        client.delete_rse_attribute(rse=rse_name, key='rule_approvers')
+        client.delete_rse_attribute(rse=rse_name, key='quota_approvers')
+    except RSEAttributeNotFound:
+        pass
+    rule_approvers = ','.join(site_managers[alt_rse])
+    print(f"Setting managers for {rse_name} to {rule_approvers}")
+
+    client.add_rse_attribute(rse=rse_name, key='rule_approvers', value=rule_approvers)
+    # For now, quota approvers are also rule approvers
+    client.add_rse_attribute(rse=rse_name, key='quota_approvers', value=rule_approvers)
+
+    # This is perhaps temporary
+    for account in site_managers[alt_rse]:
+        try:
+            client.add_account_attribute(account=account, key='country-XX', value='admin')
+        except Duplicate:
+            pass
+
+
+def get_egroup_map(all_groups, tag_relation="facility", role="local-data-manager"):
+    all_groups = make_request(CRIC_GROUP_API)
+    egroup_map = defaultdict(set)
+    for _, values in all_groups.items():
+        if values['tag_relation'] == tag_relation and values['role'] == role:
+            tag_name = values['tag_name'].lower()
+            for egroup in values['egroups']:
+                egroup_map[tag_name].add(egroup)
+
+    return egroup_map
+
+
+#############################################
+# Group identities sync
+############################################
+
+def sync_roles_to_groups(client, users):
+    """
+    Sync roles to respctive group accounts
+    - local_users: site groups
+    - group_users: physics groups
+    """
+
+    local_users = defaultdict(set)
+    group_users = defaultdict(set)
+
+    # Both can be used as one as well
+    dn_email_map_local = {}
+    dn_email_map_group = {}
+
+    site_map = build_site_facility_map()
+    all_groups = make_request(CRIC_GROUP_API)
+    # get groups for group email accounts
+    facility_egroups = get_egroup_map(all_groups, tag_relation="facility", role="local-data-manager")
+    group_egroups = get_egroup_map(all_groups, tag_relation="group", role="group-data-manager")
+
+    for user in users:
+        roles = user['ROLES']
+        dn = user['DN']
+        email = user['EMAIL']
+
+        # data-managers: are site managers / admins?
+        # We also add them to list of local users
+        if 'data-manager' in roles:
+            for tag in roles['data-manager']:
+                if tag.startswith('site:'):
+                    site = tag.replace('site:', '', 1)
+                    local_users[site].add(dn)
+                    dn_email_map_local[dn] = email
+
+        # local-data-managers: set of users that manage data for #TODO (the site)?
```

Comment:
I'd say these are the people who can place data at the site. Somewhat different from a manager.